fcrepo/fcrepo
# fix rebuild to commit transactions

**JIRA Ticket**: https://fedora-repository.atlassian.net/browse/FCREPO-3713

# What does this Pull Request do?

Fixes the rebuild process to commit the transactions that they create. This fixes the bug where resources are removed when the tx manager rollsback expired transactions.

# How should this be tested?

1. Start Fedora
2. Verify you can get the root
3. Wait a few minutes
4. Verify you can still get the root

# Interested parties
@fcrepo/committers
